nx.json

```diff
@@ -118,6 +118,9 @@
     },
     "server-repository-core-domain-services": {
       "tags": ["platform:server", "scope:repository", "type:domain-services"]
+    },
+    "server-shared-utils": {
```

Comment:
Try to not create too general libraries. Consider changing the name to something like server-shared-util-repository